src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -339,6 +339,7 @@ static constexpr Property<std::shared_ptr<ov::Model>> model{"MODEL_PTR"};
  * @brief Special key for auto batching feature configuration. Enabled by default
  */
 static constexpr Property<bool, PropertyMutability::RW> allow_auto_batching{"ALLOW_AUTO_BATCHING"};
+static constexpr Property<std::string> auto_batching_device_config{"AUTO_BATCH_DEVICE_CONFIG"};
```

Comment:
why?? we must use common property `ov::device::priorities`

Reply:
this change is not need for this PR anyway